pysat/_meta.py

```diff
@@ -477,19 +249,6 @@ def __str__(self, long_str=True):
 
         return out_str
 
-    def _insert_default_values(self, input_name):
-
-        default_str = ''
-        default_nan = np.NaN
-        labels = [self.units_label, self.name_label, self.notes_label,
-                  self.desc_label, self.plot_label, self.axis_label,
-                  self.scale_label, self.min_label, self.max_label,
-                  self.fill_label]
-        defaults = [default_str, input_name, default_str, default_str,
```

Comment:
I believe this is the issue. The default for some of the metadata parameters is set to `input_name`. When I traced through the current code I don't see this functionality replicated.